portal/portal-charon/charon/src/webapp/scripts/jumptotop.js

```diff
@@ -8,11 +8,17 @@
     };
 
     JumpToTop.prototype.init = function() {
+        portal.i18n.loadProperties({
+            resourceClass: 'org.sakaiproject.portal.api.PortalService',
+            resourceBundle: 'jumptotop',
+            namespace: 'jumptotop'
+        });
+
         this.$link = $("<a>");
         this.$link.attr("id", "jumptotop");
         this.$link.addClass("jumptotop");
         this.$link.addClass("hidden");
-        this.$link.attr("title", "Jump to top");
+        this.$link.attr("title", portal.i18n.tr('jumptotop', 'jumptotop_title'));
```

Comment:
actually, this line should be in a callback. loadProperties is async so you may get odd timing issues.

var self = this;
...
namespace: "jumptotop",
callback: function () { self.$link.attr("title", portal.i18n.tr('jumptotop', 'jumptotop_title')); }
...

Or something like that. Also, you can set debug: true as one of your options to loadProperties.

Reply:
@adrianfish - I've also noticed that when I'm not logged in and am just on /portal, I see a js error in the console:
"Uncaught TypeError: portal.i18n.loadProperties is not a function"
Is the sakai.morpheus.i18n.utils.js file not being loaded in that case?

Reply:
It is there, on the gateway. Strange error, maybe the function fails to be defined because of some logic depending on being logged in. Bad error! Another JIRA for that though, perhaps?